src/Annotation/AnnotationTable.jsx

```diff
@@ -8,11 +9,36 @@ import {
   addLayerSignalRemover,
 } from '@janelia-flyem/react-neuroglancer';
 
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import Box from '@material-ui/core/Box';
 import DataTable from './DataTable/DataTable';
 import {
-  getRowItemFromAnnotation, isValidAnnotation, getAnnotationPos, getAnnotationIcon,
+  getRowItemFromAnnotation,
+  isValidAnnotation,
+  getAnnotationPos,
+  getAnnotationIcon,
+  toAnnotationPayload,
 } from './AnnotationUtils';
 import AnnotationToolControl from './AnnotationToolControl';
+import ImportAnnotation from './ImportAnnotation';
+import ExportAnnotation from './ExportAnnotation';
+
+function debounce(func, wait, immediate) {
```

Comment:
Can we move this function into src/utils/debounce.js? It woule be useful to have it as a shared function. I can imagine it will be useful elsewhere in the code.